### PR TITLE
Yohannes taye Task submission

### PR DESCRIPTION
**Pull request by yohannestayez**

### CODE REVIEW

**Comment by yohannestayez** [comment]
Demonstrated the necessity of backpropagation by comparing the performance of a neural network with and without backpropagation. Implemented a 3 hidden-layer network on a subset of the MNIST dataset from sklearn, using ReLU activation and a softmax output layer. Used cross-entropy loss for training with backpropagation.
